app/javascript/controllers/dropdown_menu_controller.js

```diff
@@ -19,6 +16,5 @@ export default class extends Controller {
 
   toggleMenu() {
     this.menuTarget.classList.toggle('hidden')
-    console.log("hi")
```

Comment:
These changes have since been merged in another PR so you can remove that file from yours to avoid rewriting the changes